spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/datastore/GcpDatastoreAutoConfiguration.java

```diff
@@ -35,13 +35,7 @@
 import org.springframework.cloud.gcp.core.UserAgentHeaderProvider;
 import org.springframework.cloud.gcp.data.datastore.core.DatastoreOperations;
 import org.springframework.cloud.gcp.data.datastore.core.DatastoreTemplate;
-import org.springframework.cloud.gcp.data.datastore.core.convert.DatastoreCustomConversions;
-import org.springframework.cloud.gcp.data.datastore.core.convert.DatastoreEntityConverter;
-import org.springframework.cloud.gcp.data.datastore.core.convert.DatastoreServiceObjectToKeyFactory;
-import org.springframework.cloud.gcp.data.datastore.core.convert.DefaultDatastoreEntityConverter;
-import org.springframework.cloud.gcp.data.datastore.core.convert.ObjectToKeyFactory;
-import org.springframework.cloud.gcp.data.datastore.core.convert.ReadWriteConversions;
-import org.springframework.cloud.gcp.data.datastore.core.convert.TwoStepsConversions;
+import org.springframework.cloud.gcp.data.datastore.core.convert.*;
```

Comment:
@meltsufin you may help me here.. I'm using spring import options and codestyle. This should happen? Sorry for this. I can revert this change if necessary.